doc/integrator/ngeo.rst.tmpl

```diff
@@ -99,8 +99,10 @@ The sub section is the interface name, and after that we have:
     (but can not be removed via interface configuration).
 * ``constants``: Directly define an ``angular`` constant in the vars file.
 * ``dynamic_constants``: Define an ``angular`` constant from a dynamic values.
-* ``static``: key: Constant name, value: name of the static view that we want to have the URL.
-* ``routes``: key: Constant name, value:
+* ``static``, the ``key`` is the constant name:
+    ``name``: The path of the ressource witch one we want to have the URL.
```

Comment:
I don't understand this sentence

Reply:
=> https://docs.pylonsproject.org/projects/pyramid/en/1.10-branch/api/request.html#pyramid.request.Request.static_url ...

Reply:
So is it 
"the relative path of the resource for which an URL is to be provided"
?

Reply:
Usualy it's an absolute path that starts with '/etc/geomapfish/static/' it can also start with a python package name like `<package>:<path_relative_to_the_package>`but it's not a common uasge...

Reply:
I update it to:
The path of the resource witch one we want to have the URL e.g. ``/etc/geomapfish/static/contextualdata.html``.